README.md

```diff
@@ -20,23 +20,18 @@ A Python tool to assist the task of scheduling a conference which:
 * Event - a talk or workshop
 * Demand - the predicted size of audience for an event
 
-# Built-In Constraints
+# Constraints
+* All events must be scheduled
 * A slot may only have a maximum of one event scheduled
-* A room may only be scheduled to host an event for which it is deemed suitable
-* An event has at least one defined role, none of which may be 'Session Chair'
-* A person may only perform one role in any time slot except for 'Mentor' which is allowed alongside any other role
-* A person must not be scheduled to fulfil a role in a time slot for which they are unavailable
-* Any person assigned the 'Session Chair' role must not be assigned any other role in the same session
-* No person must be scheduled to chair more sessions than they have specified themselves nor the maximum number defined for the conference
-* A workshop does not require a session chair
+* An event must not be scheduled in a slot for which it has been marked as unavailable
+* An event must not be scheduled at the same time as another event for which it has been marked not to clash
+* An event may be tagged and, if so, must be scheduled in a session where it shares at least one tag with all other events in that session
 
-# Built-In Optimisations
+# Optimisation
 * The sum of 'potential disappointments' should be minimised where 'potential disappointments' is defined as the excess of demand over room capacity for every scheduled event
-* The number of slot/talk combinations without a person assigned to the 'Session chair' role should be minimised
-* The number of occasions where the session chair changes within a session for any room should be minimised
 
-# Ad-Hoc Constraints and Optimisations
-Some examples of situations which have arisen at previous conferences and should be handled by defining ad-hoc constraints or optimisations:
+# Examples
+Some examples of situations which have arisen at previous conferences and chould be handled by the unavailability, clashing and tagging constraints:
```

Comment:
@meatballs just in case you somehow miss @alexwlchan's comment because of my fast merge. :arrow_up: